backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -234,6 +236,14 @@ public void enableGLDebugOutput (boolean enable, PrintStream debugOutputStream)
 		debugStream = debugOutputStream;
 	}
 
+	/** Configuration which will be set on if the {@link Os} is {@link Os/MacOsX}.
```

Comment:
The second link is invalid; it should be `{@link Os#MacOsX}` . Don't ask me why the syntax is weird and different, I don't know!